### PR TITLE
[RS-204] Send Github Events to Discord channel

### PR DESCRIPTION
**Pull request by LucasLeguizamo**
## Changes Made 🎉

- [x] feat: added new feature to improve user experience
- [ ] fix: corrected a bug with login functionality
- [ ] refactor: improved code readability and organization
- [ ] docs: updated README with new instructions
- [ ] chore: updated dependencies and configuration files

### Describe Changes
I added a notification system for discord when an issue and PR are open/closed 
## Related Issue(s)

- [204](https://github.com/serudda/reward-system/issues/204)

## Visuals (Optional)

image
image

## Checklist ✅

- [x] My code follows the code style of this project.

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by LucasLeguizamo** [comment]
image

File: apps/nextjs/src/common/types/githubEvent.ts [hunk 1]
```diff
@@ -0,0 +1,213 @@
+export enum IssueActions {
+  opened = 'opened',
+  closed = 'closed',
+  edited = 'edited',
+}
+
+export enum IssueColors {
+  opened = '238636',
```
Comment (serudda) [inline apps/nextjs/src/common/types/githubEvent.ts:8]: Buen color 🫰

File: apps/nextjs/src/common/types/index.ts [hunk 2]
```diff
@@ -1 +1,2 @@
 export * from './input';
+export * from './githubEvent';
```
Comment (serudda) [inline apps/nextjs/src/common/types/index.ts:2]: Buen import ahi... fachero

File: apps/nextjs/src/pages/api/github.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        author: {
+          name: '',
+          icon_url: '',
+          url: '',
+        },
+      };
+
+      if (issue) {
+        embed = {
+          ...embed,
```
Comment (serudda) [inline apps/nextjs/src/pages/api/github.ts:31]: buen destructiring...

File: packages/api/src/router/bot.ts [hunk 4]
```diff
@@ -1,9 +1,12 @@
-import { sendDiscordMsgHandler } from '../controllers/bot.controller';
-import { sendDiscordMsgInput } from '../schema/bot.schema';
+import { sendDiscordMsgHandler, sendDiscordRewardMsgHandler } from '../controllers/bot.controller';
+import { sendDiscordGithubMsgInput, sendDiscordMsgInput } from '../schema/bot.schema';
 import { createTRPCRouter, publicProcedure } from '../trpc';
 
 export const botRouter = createTRPCRouter({
```
Comment (serudda) [inline packages/api/src/router/bot.ts:5]: Buen uso de TRPC 🥰

**Review by serudda — APPROVED** [review]

**Pull request merged**